app/src/main/java/a3/com/convo/activities/ProfileActivity.java

```diff
@@ -12,9 +12,6 @@
 
 public class ProfileActivity extends AppCompatActivity {
 
-    private AddInfoFragment addInfoFrag;
-    private ProfileDetailsFragment profileDetailsFrag;
-    private Fragment profileFrag;
     private FragmentTransaction ft;
```

Comment:
We dont need to save FragmentTransaction in memory. We can always get it from getSupportFragmentManager()